### DISCUSSION TITLE
Error to upgrade to 3.2 from 3.0

### DISCUSSION THREAD

**Discussion by DeivisFelipe** [nuxt/nuxt#19587] [General]
I am trying to upgrade to 3.2 version from 3.0, but i get the following error

![image](https://user-images.githubusercontent.com/36674115/224366289-2d6638c0-14a7-421d-a400-ed2d38649d44.png)

**Comment by danielroe** [comment]
Apologies - this is a very recent regression in a dependency - track the fix in https://github.com/unjs/hookable/pull/71
